android/help/build-help.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+# Set sensible script defaults:
+# set -e: Terminate script if a command returns an error
+set -e
+# set -u: Terminate script if an unset variable is used
+set -u
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+QUIET=0
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+THIS_DIR="$(dirname "$THIS_SCRIPT")"
+
+display_usage() {
+  echo "build.sh [--no-clean] target [...target]"
+  echo "Builds help documentation for Keyman for Android"
+  echo "Targets:"
+  echo "  * htm: convert documentation to html using pandoc"
+  echo
+  echo " --no-clean: don't clean target folder before building"
+}
+
+DO_HTM=false
+DO_CLEAN=true
+
+# Debug flags
+DO_HTM_CONVERSION=true
+
+#
+# Parse args
+#
+
+shopt -s nocasematch
+
+while [[ $# -gt 0 ]] ; do
+  key="$1"
+  case $key in
+    htm)
```

Comment:
Why `htm` and not `html`? Given the generated files have the .html extension?

Reply:
I was replacing `chm` to `htm` from the original script. I can tweak to accept both `htm` and `html` :)